src/main/geometries/Geometries.java

```diff
@@ -0,0 +1,50 @@
+package geometries;
+
+import java.util.LinkedList;
+import java.util.List;
+import primitives.Point;
+import primitives.Ray;
+
+public class Geometries implements Intersectible {
+
+	private List<Intersectible> intersectibles = new LinkedList<>();
```

Comment:
Why a linkedList?

Reply:
Because insertion is O(1) as opposed to O(n) in an arraylist (in the case where it needs to be resized). We don't need direct access, just iteration.